src/dev/GraphiQL.tsx

```diff
@@ -0,0 +1,218 @@
+import React from "react";
+import { GraphiQL as DefaultGraphiQL } from "graphiql";
+import "graphiql/graphiql.css";
+import { createSocketIOGraphQLClient } from "@n1ru4l/socket-io-graphql-client";
+import { useSocket } from "../socket";
+import { buildApiUrl } from "../public-url";
+import useAsyncEffect from "@n1ru4l/use-async-effect";
+import { RelayEnvironmentProvider } from "relay-hooks";
+import { useStaticRef } from "../hooks/use-static-ref";
+import { usePersistedState } from "../hooks/use-persisted-state";
+import { SplashScreen } from "../splash-screen";
+import { AuthenticationScreen } from "../authentication-screen";
+import { Socket as IOSocket } from "socket.io-client";
+import { createEnvironment } from "../relay-environment";
+import { useLogInMutation } from "../chat/log-in-mutation";
+
+const usePcPassword = () =>
```

Comment:
Since this is a 1:1 copy: can you move the hook to the `hooks/use-pc-password.tsx` file and update all usages to import it from there? :)